js/deer-config.js

```diff
@@ -3,6 +3,7 @@ export default {
     TYPE: "deer-type",              // attribute, JSON-LD @type
     TEMPLATE: "deer-template",      // attribute, enum for custom template
     KEY: "deer-key",                // attribute, key to use for annotation
+    KEYDUPLICATE: "deer-key-duplicate", // attribute, notes input is a duplicate and app should ignore       
```

Comment:
Yes.  This is the value DEER will write to inputs that it detects are duplicates, pretty much just as a warning for the dev that it is going to be ignored.  It doesn't have to be a config, or even necessarily written (as you said, in these situations we could just change the DEER.KEY attribute so it will be ignored).  This just seemed to me to be the most obvious and hand-holdy.